PowerShellGet/PSModule.psm1

```diff
@@ -4906,6 +4906,16 @@ function Test-ScriptFileInfo
             return
         }
 
+        if ($PSScriptInfo.Version -match '-')

```

Comment:
this will match even if there are more than 1 hyphens like  "Mymodule.1.2.3-prerelease-alpha.1" -match '-'. Do you want that?